models/database.go

```diff
@@ -542,6 +542,25 @@ var databaseSchema = [][]string{
 			ADD COLUMN schedule_id VARCHAR`,
 		`ALTER TABLE artifacts ALTER COLUMN type DROP DEFAULT`,
 	},
+	41: {
+		`CREATE TYPE software_version AS (
+  			name VARCHAR NOT NULL CHECK (name <> ''),
+  			version VARCHAR NOT NULL CHECK (version <> '')
+		);`,
+		`CREATE TABLE service_software_versions (
+			id VARCHAR NOT NULL,
+			service_id VARCHAR NOT NULL CHECK (service_id <> ''),
+			software_versions ARRAY software_version,
```

Comment:
looks like ARRAY is not supported by the reform lib, so probably need to change to JSONB

Reply:
in docs i see:
```
pay_by_quarter  integer ARRAY,
```